templates/microcloud/resources.html

```diff
@@ -53,10 +52,65 @@ <h3 class="p-heading--5 u-no-padding--top u-no-margin--bottom">
               <a href="https://ubuntu.com/engage/introduction-to-micro-clouds">Introduction to MicroClouds</a>
             </h3>
             <p>
-              A micro cloud is a new class of infrastructure for on-demand computing at the edge. In this webinar we go over what MicroClouds are and why they should be a part of your edge strategy, finishing with a demo.
+              MicroCloud is the newest addition to Canonical’s cloud infrastructure portfolio. It is a low-touch, open-source cloud solution purpose-built for scalable clusters and edge deployments for all types of enterprises. In this webinar, we go over what MicroClouds are and why they should be a part of your edge strategy, finishing with a demo.
+            </p>
+          </div>
+
+          <div class="col-3 col-medium-2 p-section--shallow">
+            <div class="u-embedded--image">
+              {{ image(url="https://assets.ubuntu.com/v1/ec34de89-microcloud-lts.png",
+                            alt="",
+                            width="568",
+                            height="315",
+                            hi_def=True,
+                            loading="auto|lazy") | safe
+              }}
+            </div>
+            <h3 class="p-heading--5 u-no-padding--top u-no-margin--bottom">
+              <a href="https://www.youtube.com/watch?v=M0y0hQ16YuE">MicroCloud LTS Demo</a>
+            </h3>
+            <p>
+              Canonical MicroCloud's 2.1.0 LTS release features support for single-node deployments, improved security posture, and more flexibility during the initialization process. In this video, we guide you through the new process, starting with deploying a MicroCloud on a single node, and then expanding with two additional members for a highly available cluster.
             </p>
           </div>
+
           <div class="col-3 col-medium-2 p-section--shallow">
+            <div class="u-embedded--image">
+              {{ image(url="https://assets.ubuntu.com/v1/a6c813e5-vdi-with.png",
+                            alt="",
+                            width="568",
+                            height="316",
+                            hi_def=True,
+                            loading="auto|lazy") | safe
+              }}
+            </div>
+            <h3 class="p-heading--5 u-no-padding--top u-no-margin--bottom">
+              <a href="">VDI with MicroCloud and HP Anyware</a>
```

Comment:
The link is missing here

Reply:
Was not provided in the copy doc. Have asked the requester for clarification

Reply:
@eliman11 could you please help in getting us the link here.